interface/src/app/node/node.js

```diff
@@ -299,4 +307,123 @@ angular.module('adage.node', [
       };
     }
   };
-}]);
+}])
+
+.directive('enrichedGenesets', ['MathFuncts', 'PickledGenesetsService', '$log',
+  function(MathFuncts, PickledGenesetsService, $log) {
+    return {
+      templateUrl: 'node/enriched_genesets.tpl.html',
+      restrict: 'E',
+      scope: {
+        organism: '=',
```

Comment:
Since you are not changing **organism** inside the directive, you can gain some efficiency by using **@**.